chromium_src/chrome/browser/ui/webui/settings/settings_ui.cc

```diff
@@ -14,8 +14,11 @@
 
 #if BUILDFLAG(IS_WIN) && BUILDFLAG(ENABLE_BRAVE_VPN)
 #include "brave/browser/ui/webui/settings/brave_settings_secure_dns_handler.h"
+#include "brave/browser/ui/webui/settings/brave_vpn/brave_vpn_handler.h"
+#include "chrome/browser/ui/webui/settings/safety_hub_handler.h"
 
 #define SecureDnsHandler BraveSecureDnsHandler
+#define SafetyHubHandler SafetyHubHandler>(profile));AddSettingsPageUIHandler(std::make_unique<BraveVpnHandler
```

Comment:
Why not adding `BraveVpnHandler` from `BraveSettingsUI::BraveSettingsUI()`?

Reply:
Nice find, moved, thanks!